client/my-sites/menus/menus-revert-button.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const debug = debugFactory( 'calypso:menus:revert-button' ); // eslint-disable-line no-unused-vars
+
+const MenuRevertButton = React.createClass( {
+	componentWillMount() {
+		this.props.menuData.on( 'saved', this.setReverted );
+		this.props.menuData.on( 'error', this.setReverted );
+	},
+
+	componentWillUnmount() {
+		this.props.menuData.off( 'saved', this.setReverted );
+		this.props.menuData.off( 'error', this.setReverted );
+	},
+
+	getInitialState() {
+		return { reverting: false };
+	},
+
+	setReverting() {
```

Comment:
How do you think names like `startReverting()` and `endReverting()` would semantically compare to their `set*` equivalents?